docs/ibis-for-dplyr-users.ipynb

```diff
@@ -0,0 +1,777 @@
+{
```

Comment:
<p>What is the guessing algorithm in dplyr?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ibis-project/ibis/pull/6050/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='57'/>

Reply:
<p>I just looked and I think I'll at least update the language here. When the join predicate is missing, dplyr does a natural join so it's not "guessing" which is what I thought.</p>

Reply:
We could potentially implement the same natural join semantics which I think is "join on columns with the same name", but not sure if it's worth the trouble right now.